build-system/server/app-utils.js

```diff
@@ -14,31 +14,60 @@
  * limitations under the License.
  */
 
+const log = require('fancy-log');
 const minimist = require('minimist');
+const {cyan, green} = require('ansi-colors');
+
+let serveMode = 'default';
 
 /**
- * Determines the server's mode based on command line arguments.
+ * Returns a string representation of the server's mode.
  * @return {string}
  */
 function getServeMode() {
-  const argv = minimist(process.argv.slice(2), {string: ['rtv']});
-  if (argv.compiled) {
-    return 'compiled';
-  }
-  if (argv.cdn) {
-    return 'cdn';
+  return serveMode;
```

Comment:
Is this needed anymore?

Reply:
Yes, because there are several [call sites](https://github.com/ampproject/amphtml/search?utf8=%E2%9C%93&q=getServeMode) across all server code.